spec/requests/providers/means/identify_types_of_incomes_controller_spec.rb

```diff
@@ -152,7 +140,7 @@
 
         it "redirects to the income summary index page" do
           request
-          expect(response).to redirect_to(providers_legal_aid_application_income_summary_index_path(legal_aid_application))
+          expect(response).to redirect_to(providers_legal_aid_application_means_student_finance_path(legal_aid_application))
```

Comment:
ditto: redirect to changed but description hasn't

Reply:
done